src/test/java/step_definitions/LinkedinPageSteps.java

```diff
@@ -0,0 +1,102 @@
+package step_definitions;
+
+import core.Hooks;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import helpers.PropertyReader;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pages.LinkedinPage;
+
+import static core.BaseClass.*;
+
+
+/**
+ * Created by rnadir on 12/14/17.
+ */
+public class LinkedinPageSteps {
+
+    private static PropertyReader props = new PropertyReader();
+    private WebDriver driver = Hooks.getDriver();
+    private static Logger logger = LoggerFactory.getLogger(LinkedinPageSteps.class);
+    private LinkedinPage linkedinPage = PageFactory.initElements(driver, LinkedinPage.class);
+
+    public LinkedinPageSteps() throws Exception {}
+
+    @Then("^Login to \"([^\"]*)\" as \"([^\"]*)\"$")
+    public void loginToAs(String page, String user) throws InterruptedException {
+        checkTitleEquals("LinkedIn: Log In or Sign Up");
+        if (user.equalsIgnoreCase("Rufat Nadir")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "Rufat Nadir" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AS1132)